.gitignore

```diff
@@ -105,3 +106,4 @@ KML/res*centers.kml
 
 # Generated files
 src/h3lib/include/h3api.h
+src/apps/benchmarks/benchmarkCountries.c
```

Comment:
I guess it can go into the build directory - the nice thing about having it here is that it's a bit easier to edit, e.g. if your IDE is eliding `build`

Reply:
My understanding is that like `h3api.h`, it would get placed in the build directory if you are doing an out-of-source build, and this was just in case you're doing an in-source build.